livekit-plugins/livekit-plugins-nltk/setup.py

```diff
@@ -46,7 +46,11 @@
     license="Apache-2.0",
     packages=setuptools.find_namespace_packages(include=["livekit.*"]),
     python_requires=">=3.9.0",
-    install_requires=["livekit >= 0.9.2", "nltk >= 3, < 4"],
+    install_requires=[
+        "livekit >= 0.9.2",
```

Comment:
also use for livekit ~= 
otherwise this will break use code when adding breaking changes..